### ISSUE TITLE
Some clickable elements lack correct attributes

### ISSUE THREAD

**Issue by torgeirl** [devilry/devilry-django#1180]
Success Criteria 1.3.1 in WCAG 2 fails due to some clickable elements not being correctly interpreted as links:

* `... or select files`  in `devilry_group/student/XXXXXX/feedbackfeed/` (handled by the `cradmin-legacy-bulkfileupload-advanced-fileselectbutton-wrapper` class)

* `I should have had more roles` on the role selection page (set by the `DEVILRY_LACKING_PERMISSIONS_URL` setting and handled by the `col-sm-6 devilry-frontpage-more-roles-link-wrapper` class)

I suspect the same is true for the `Superuser UI` on the role selection page (handled by the `devilry-frontpage-superuser-link-wrapper` class)

**Comment by espenak** [comment]
@torgeirl I can not understand how `I should have more roles` or `Superuser UI` links can be interpreted as not being links? They are very simple `<a href="SOMEURL">Some label</a>` tags without any javascript or anything special.

**Comment by espenak** [comment]
@torgeirl I assume this is about link and button styling. I suggest we just underline all links, and re-style the `... or select files` to look like a button.

**Comment by torgeirl** [comment]
@espenak: I should have been more clear that this isn't an issue for screen readers, but rather the visual presentation. One example given by the evaluators were how the upload link lacks the look of a link as you can see here:

<img width="481" alt="tydeligere-markering-av-lenke" src="https://user-images.githubusercontent.com/796053/151534406-1ec3c9c4-bfd2-40e7-abf2-19bd55868df2.png">

**Comment by espenak** [comment]
It is fixed @torgeirl . I should have included a screenshot!

![image](https://user-images.githubusercontent.com/84501/151793676-22a21eb5-8818-4e95-b947-537606cd43a7.png)

![image](https://user-images.githubusercontent.com/84501/151793733-29c3b96a-0db2-43ab-b074-97d8a86a23e3.png)
